### DISCUSSION TITLE
Error : Invalid event name - cannot identify event

### DISCUSSION THREAD

**Discussion by ifthikar-ahmed** [getPraxis/FELauncher#5] [Q&A] [answered]
Component was working fine for  while and suddenly started getting the error.

Any idea , what is causing this ?

Error : Invalid event name - cannot identify event

**Comment by ohadios** [comment, marked as answer]
Hello Ahmed,
This error happens when you forget to add the "FlowEventListener" component to the Lightning Page.
If nothing on the page knows to listen to the "e.praxis:FlowEvent" event, then this error is shown on the page.

Could it be that someone removed the component from the page?

**Reply by ifthikar-ahmed** [reply]
Yes , exactly..

Thank you
